src/reason-parser/reason_parser.mly

```diff
@@ -2001,6 +2003,8 @@ mark_position_cl
   | let_bindings SEMI class_expr_lets_and_rest
     { class_of_let_bindings $1 $3 }
   | object_body { mkclass (Pcl_structure $1) }
+  | LET? OPEN override_flag as_loc(mod_longident) SEMI class_expr_lets_and_rest
```

Comment:
Is there a reason why we still need to support `let open ...` vs just `open ...` in newly added syntax? I tend to favour just `open` in favour of less grammar complexity. Thoughts?